getting_started/step_by_step/your_first_game.rst

```diff
@@ -865,49 +843,60 @@ Note that a new instance must be added to the scene using ``add_child()``.
 
     func _on_MobTimer_timeout():
         # Choose a random location on Path2D.
-        $MobPath/MobSpawnLocation.offset = randi()
+        var mob_spawn_location = get_node("MobPath/MobSpawnLocation");
+        mob_spawn_location.offset = randi()
+
         # Create a Mob instance and add it to the scene.
-        var mob = Mob.instance()
-        add_child(mob)
+        var mob_instance = _mob_scene.instance()
+        add_child(mob_instance)
+
         # Set the mob's direction perpendicular to the path direction.
-        var direction = $MobPath/MobSpawnLocation.rotation + PI / 2
+        var direction = mob_spawn_location.rotation + TAU / 4
+
         # Set the mob's position to a random location.
-        mob.position = $MobPath/MobSpawnLocation.position
+        mob_instance.position = mob_spawn_location.position
+
         # Add some randomness to the direction.
-        direction += rand_range(-PI / 4, PI / 4)
-        mob.rotation = direction
-        # Set the velocity (speed & direction).
-        mob.linear_velocity = Vector2(rand_range(mob.min_speed, mob.max_speed), 0)
-        mob.linear_velocity = mob.linear_velocity.rotated(direction)
+        direction += rand_range(-TAU / 8, TAU / 8)
+        mob_instance.rotation = direction
+
+        # Choose the velocity.
+        mob_instance.linear_velocity = Vector2(rand_range(mob_instance.min_speed, mob_instance.max_speed), 0).rotated(direction)
```

Comment:
This results in a much longer and harder to read line. We originally had it written like that in an early version of the game, and this led to additional confusion, which was why it was broken into multiple lines.

Reply:
Good point, I changed this to be split into two lines, but I think it's better to use a local field rather than storing the temporary value inside of `linear_velocity`.